pkgs/development/libraries/polkit/default.nix

```diff
@@ -12,19 +33,23 @@
 }:
 
 let
-
+  boolToString = b: if b then "true" else "false";
```

Comment:
Is it possible to use [`lib.boolToString`](https://github.com/NixOS/nixpkgs/blob/c9b7cc79e6ce6b38debb28ab0d06051a4fb5b336/lib/trivial.nix#L106-L114) instead?

Reply:
Thanks, was looking at the wrong library module so I did not find this.